src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/StandardPlayerRequestResolver.java

```diff
@@ -80,6 +85,30 @@ public void resolve(@NotNull final IRequestManager manager, @NotNull final IRequ
         final IColony colony = manager.getColony();
         if (colony instanceof Colony)
         {
+            if (Configurations.requestSystem.creativeResolve &&
+                  request.getRequest() instanceof IDeliverable &&
+                  request.getRequester() instanceof BuildingBasedRequester &&
+                  ((BuildingBasedRequester) request.getRequester()).getBuilding().isPresent() &&
+                  ((BuildingBasedRequester) request.getRequester()).getBuilding().get() instanceof AbstractBuilding)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "Optional#isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3655)